omnibot/processor.py

```diff
@@ -41,31 +46,38 @@ def process_event(event):
         bot.logging_context,
     )
     statsd.incr(f"event.process.attempt.{event_type}")
-    if event_type == "message" or event_type == "app_mention":
+    if event_type in {"message", "app_mention", "reaction_added", "reaction_removed"}:
         try:
             with statsd.timer("process_event"):
                 logger.debug(
-                    f"Processing message: {json.dumps(event, indent=2)}",
+                    f"Processing event: {json.dumps(event, indent=2)}",
                     extra=event_trace,
                 )
-                try:
-                    message = Message(bot, event_info, event_trace)
-                    _process_message_handlers(message)
-                except MessageUnsupportedError:
-                    pass
+                if event_type == "message" or event_type == "app_mention":
+                    try:
+                        message = Message(bot, event_info, event_trace)
+                        _process_message_message_handlers(message)
+                    except MessageUnsupportedError:
+                        pass
+                elif event_type == "reaction_added" or event_type == "reaction_removed":
```

Comment:
why nest it?

Reply:
wasn't it clearer before with

messages
  --> process message
reactions
  --> process reactions?